setup/features/notifications/index.md

```diff
@@ -25,7 +25,13 @@ Additionally, Spinnaker is capable of handling cron-based triggers and detect ch
 
 # Configuring Notifications
 
-Notification configurations are set in echo.yml and settings.js. On a single instance virtual machine installation, notification values can also be set in spinnaker-local.yml, which will pass the details to echo.yml and settings.js.
+Notification configurations are in echo.yml and settings.js. For changes to echo.yml, create echo-local.yml and put your changes in there. You can customize settings.js directly.
+
+## Where to put echo-local.yml and settings.js
```

Comment:
Since you added this subheading, I would add another beneath the text you added to clarify that this section is over.